### ISSUE TITLE
time Dp != time Dp?

### ISSUE THREAD

**Issue by Pittini** [rg-engineering/ioBroker.heatingcontrol#85]
If i write a time value via JS setState in the Field OverrideTime, everything is fine (cause its defined as string).
If i try to do exactly the same with one of the time fields, z.B. (heatingcontrol.0.Profiles.0.Bad.Mo-Fr.Periods.0.time), i got the following Warn Message:

You are assigning a string to the state "heatingcontrol.0.Profiles.0.Bad.Mo-Fr.Periods.0.time" which expects a date. Please fix your code to use a date or change the state type to string. 

Also if i try to convert the string like "00:00" to date objekt i got similar Warnings

I've tried several methods, but all are failing:

1.
```
/* Geht nicht
var d = new Date("1970.1.1" + " " + "05:00");
log(d.getTime())
let dummy = formatDate(d.getTime(), "hh:MM");
log(dummy)
log(typeof(dummy))
setState("heatingcontrol.0.Profiles.0.Bad.Mo-Fr.Periods.0.time", dummy)
*/

```

2.
```
/* Geht nicht
let d=new Date()
d.setHours(5,0)
log(formatDate(d.getTime(), "hh:MM"))
setState("heatingcontrol.0.Profiles.0.Bad.Mo-Fr.Periods.0.time", formatDate(d.getTime(), "hh:MM"))
*/
```
3.
```
/* Geht auch nicht
let dummy = getDateObject('00:00');
log(formatDate(dummy, "hh:MM"))
setState("heatingcontrol.0.Profiles.0.Bad.Mo-Fr.Periods.0.time", formatDate(dummy, "hh:MM"))
*/

```

**Comment by rg-engineering** [comment]
see issue #87

**Comment by rg-engineering** [comment]
solved with 0.3.19
